client/me/account-close/closed.scss

```diff
@@ -4,7 +4,7 @@
 
 	.account-close__spinner-text {
 		font-size: 14px;
-		font-weight: 300;
+		font-weight: 400;
```

Comment:
The intent of this was to make it like the Customizer loading screen. 

<img width="644" alt="Screenshot 2020-03-07 at 14 34 29" src="https://user-images.githubusercontent.com/43215253/76145367-c5de9300-6080-11ea-9710-780735fe194e.png">

That doesn't use a `font-weight`, so maybe it's better to drop this line completely?

Reply:
@Aurorum Isn't `font-weight: 400;` the default font weight, thus, keeping or dropping the line should lead to the same result, if that element does not inherit its font weight from one of its parent elements.

Reply:
I was comparing it with the default, and I could've sworn there was a difference, but perhaps my eyes were playing tricks on me! :)

Reply:
Thus, all good, @Aurorum, right?